.github/workflows/release.yml

```
@@ -0,0 +1,75 @@
+name: Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Set up Python
+        uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
+        with:
+          python-version: 3.12
+
+      - name: Build package using ros_release_python
+        run: |
+          # Create a temporary directory for ros_release_python
+          TEMP_DIR=$(mktemp -d)
+          cd $TEMP_DIR
+
+          # Clone the ros_release_python repository
+          git clone https://github.com/ros-infrastructure/ros_release_python.git
+
+          # Build the docker image
+          cd ros_release_python
+          docker build -t rrp .
+
+          # Get the repository name
+          REPO_NAME=$(basename $GITHUB_REPOSITORY)
+
+          # Run the docker container to build the package
+          docker run --rm \
+            -v "$GITHUB_WORKSPACE:/projects/$REPO_NAME" \
+            -v "$(pwd):/ros_release_python" \
+            -w "/projects/$REPO_NAME" \
+            rrp \
+            bash -c "/ros_release_python/scripts/ros_release_python pip"
+
+      - name: Upload assets to release
+        run: |
+          gh release upload ${{ github.event.release.tag_name }} dist/* --clobber
```

```suggestion
          gh release upload ${{ github.event.release.tag_name }} dist/*
```

This IMO made sense for testing but should be dropped for the prod workflow.